Source/MetroXHunter/HUD/MainHUD.h

```diff
@@ -19,7 +19,7 @@ class UMainHUD : public UInterface
 };
 
 /**
- *
+ * Main HUD interface to be able to update the viewport.
```

Comment:
Let's put the "type" keyword first (here the "interface"), for example "Interface of the main HUD to update the viewport."